core/services/ocr2/delegate.go

```diff
@@ -714,6 +718,15 @@ func (d *Delegate) newServicesGenericPlugin(
 		if ocr3Provider, ok := provider.(types.OCR3ContractTransmitter); ok {
 			contractTransmitter = ocr3Provider.OCR3ContractTransmitter()
 		}
+		var onchainKeyringAdapter ocr3types.OnchainKeyring[[]byte]
+		if onchainSigningStrategy.IsMultiChain() {
+			onchainKeyringAdapter, err = ocrcommon.NewOCR3OnchainKeyringMultiChainAdapter(d.ks, &onchainSigningStrategy, lggr)
```

Comment:
@vyzaldysanchez We should extract the config here and transform it into a map[string]ocr2KeyBundle. before passing it into the multichain adapter.